### PR TITLE
Add new HUD style by monochrome9090

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

Port of https://github.com/Liberty-Landing/Liberty-Station-13/pull/503 with an additional fix from me.

image

Normal:
image

Minimized:
image

## Why It's Good For The Game

More customization and rad sprites is better.

## Testing

Launched local server, switched styles back and forth, changed to minimized variant and back.
Pushed HUD buttons such as "throw", "pull" and others.
Committed drug abuse and self-harm to see that HUD indicators change properly. Then did the same in-game, all works.

## Changelog
:cl:
add: "LibertyStyle" HUD variant
/:cl:

### CODE REVIEW

File: code/_onclick/hud/screen_objects/base_screen_objects.dm [hunk 1]
```diff
@@ -455,6 +455,83 @@
 	H.nano_ui_interact(H)
 	return	TRUE
 
+/obj/screen/sanity_alt
+	name = "inspiration"
+	desc = "The color of the icon displays how close you are to receive an epiphany from your experiences.\
+	Once your inspiration is high enough, you will crave for food, drinks or drugs, to be able to reflect upon all you've learned so far. \
+	Satisfy these cravings and you'll be able to \"rest\", improving upon your stats slightly, or depending on any anomalies held, \
+	you'll gain a new perk, for ill or good, and better stat gains."
```
Comment (SirRichardFrancis) [inline code/_onclick/hud/screen_objects/base_screen_objects.dm:463]:
```suggestion
	desc = "The color of the icon displays how close you are to receive an inspiration from your experiences.\
	Once you gain an inspiration, you will crave for food, drinks or drugs, to be able to reflect upon all you've learned so far. \
	Satisfy these cravings and you'll be able to \"rest\", improving upon your stats slightly, or depending on any oddities held, \
	you'll gain a new perk, for ill or good, and better stat gains."
```

**Comment by Firefox13** [comment]
> Pushed HUD buttons such as "throw", "pull" and others.

What about implant/gear info buttons?

**Review by Firefox13 — COMMENTED** [review]
I find some of the design choices really odd (the hell are those icons near the bottom left? and the info HUD being all over the place) but I have nothing against it personally. Except that brain sanity icon looks messed up but eh.

**Comment by SirRichardFrancis** [comment]
> What about implant/gear info buttons?

image

> I find some of the design choices really odd (the hell are those icons near the bottom left?

Green is NSA and red is nutriments, they are located right near the health indicator.

> and the info HUD being all over the place)

You can stick to old HUD if you don't like how this one is structured.

> but I have nothing against it personally. Except that brain sanity icon looks messed up but eh.

It goes from smooth to normal, depending on insight level.

**Comment by SirRichardFrancis** [comment]
An image from downstream:
image

**Comment by MLGTASTICa** [comment]
poopendarken.

**Comment by Holvr0** [comment]
I like it :D Variants and customisation options are cool.

**Review by Firefox13 — APPROVED** [review]
> You can stick to old HUD if you don't like how this one is structured.

With this reasoning we can add sixteen more bad HUDs and use these words when someone questions why we have so many low-quality options (this is not the case).

This reminds me of probable issue where prefs for HUDs are not being saved at all...

**Comment by SirRichardFrancis** [comment]
> With this reasoning we can add sixteen more bad HUDs and use these words when someone questions why we have so many low-quality options (this is not the case).

Point is that not everyone have to like new HUD over any other, it's true with any feature (since it's virtually impossible to have _everyone_ agree on any given PR), but HUD variants especially so, because player can choose not to switch to new HUD and play as if this PR never happened.

Besides, if this HUD is bad, why approve it?

> This reminds me of probable issue where prefs for HUDs are not being saved at all...

HUD style is being saved properly and applied after server restart just fine.
What isn't being saved exactly? Color? Transparency? Minimized status?

**Comment by Firefox13** [comment]
> Besides, if this HUD is bad, why approve it?

I didn't say it was bad.

> Point is that not everyone have to like new HUD over any other

It has to pass atleast basic quality check. This HUD is fine.

> What isn't being saved exactly? Color? Transparency? Minimized status?

Minimized status, if I recall correctly.

**Comment by Catalaria** [comment]
> > Besides, if this HUD is bad, why approve it?
> 
> I didn't say it was bad.
> 
> > Point is that not everyone have to like new HUD over any other
> 
> It has to pass atleast basic quality check. This HUD is fine.
> 
> > What isn't being saved exactly? Color? Transparency? Minimized status?
> 
> Minimized status, if I recall correctly.

Artist here. 
If you believe it has not passed the quality check, dare I ask for proper criticism?

**Comment by MLGTASTICa** [comment]
> > > Besides, if this HUD is bad, why approve it?
> > 
> > 
> > I didn't say it was bad.
> > > Point is that not everyone have to like new HUD over any other
> > 
> > 
> > It has to pass atleast basic quality check. This HUD is fine.
> > > What isn't being saved exactly? Color? Transparency? Minimized status?
> > 
> > 
> > Minimized status, if I recall correctly.
> 
> Artist here. If you believe it has not passed the quality check, dare I ask for proper criticism?

1)Blood focused (Eris is set around Sci-fi cyberpunk)
2)Elements which arent explainitive by self (health and hunger will get confused, NSA is confusing) The rest is fine in my opinion.

**Comment by ValoTheValo** [comment]
variety is the spice of life

**Pull request merged**